cmd/web/doctor_signup_form.templ

```diff
@@ -0,0 +1,241 @@
+package web
+
+templ DoctorSignupFormPage(doctorSignupHandleEndpoint string) {
+	@Base() {
+		<div id="hello-container">
+			@DoctorSignupFormScreen(doctorSignupHandleEndpoint)
+		</div>
+	}
+}
+
+templ DoctorSignupFormScreen(doctorSignupHandleEndpoint string) {
+    <!--
+  This example requires some changes to your config:
+  
+  ```
+  // tailwind.config.js
+  module.exports = {
+    // ...
+    plugins: [
+      // ...
+      require('@tailwindcss/forms'),
+    ],
+  }
+  ```
+-->
+<div class="flex min-h-full flex-col justify-center py-12 sm:px-6 lg:px-8">
+  <div class="sm:mx-auto sm:w-full sm:max-w-md">
+    <img class="h-30 w-auto max-w-full max-h-full object-contain" src="/assets/images/ordindia.png" alt=""/>
+  </div>
+
+  <div class="space-y-10 divide-y divide-gray-900/10">
+    <div class="mt-8 space-y-10 divide-y divide-gray-900/10">
+      <form action={ templ.SafeURL(doctorSignupHandleEndpoint)} method="POST" onsubmit="return confirmPassword()">
+        <div class="grid grid-cols-1 gap-x-8 gap-y-8 pt-10 md:grid-cols-3">
+          <div class="px-4 sm:px-0">
+            <h2 class="text-base font-semibold leading-7 text-gray-900">Personal Information</h2>
+            <p class="mt-1 text-sm leading-6 text-gray-600">Please provide your personal and contact details. </p>
+          </div>
+
+          <div class="bg-white shadow-sm ring-1 ring-gray-900/5 sm:rounded-xl md:col-span-2">
+            <div class="px-4 py-6 sm:p-8">
+              <div class="grid max-w-2xl grid-cols-1 gap-x-6 gap-y-8 sm:grid-cols-6">
+                <div class="sm:col-span-3">
+                  <label for="first_name" class="block text-sm font-medium leading-6 text-gray-900">
+                    First name <span class="text-red-600">*</span>
+                  </label>
+                  <div class="mt-2">
+                    <input type="text" name="first_name" id="first_name" autocomplete="given-name" required class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6">
```

Comment:
Please add the class `p-1` to all the input fields as done in the doctors page to fix the padding the issues